guide/guide/.js/src/main/scala/io/udash/web/guide/views/ext/demo/bootstrap/ListGroupDemo.scala

```diff
@@ -10,16 +10,18 @@ object ListGroupDemo extends AutoDemo with CssView {
   private val (rendered, source) = {
     import io.udash._
     import io.udash.bootstrap.list.UdashListGroup
+    import io.udash.bootstrap.utils.BootstrapStyles
     import io.udash.bootstrap.utils.BootstrapStyles._
     import io.udash.css.CssStyleName
     import org.scalajs.dom.window
+    import scalatags.JsDom.all._
 
     val news = SeqProperty("Title 1", "Title 2", "Title 3")
 
     def newsStyle(newsProperty: Property[String]): ReadableProperty[CssStyleName] = {
       newsProperty.transform(_.last match {
         case '1' => active
-        case '2' => disabled
+        case '2' => BootstrapStyles.disabled
```

Comment:
You might consider renamed import:
```scala
import io.udash.bootstrap.utils.BootstrapStyles.{disabled => disabledStyle, _}
```
and then using `disabledStyle` without `BootstrapStyles` prefix and import. Also, `BootstrapStyles#active` could be imported as `activeStyle` for consistency although it's not required.